src/main/java/betterquesting/client/gui2/editors/nbt/PanelScrollingNBT.java

```diff
@@ -51,23 +66,25 @@ public class PanelScrollingNBT extends CanvasScrolling implements IPEventListene
 
     private final Stack<NBTBase> nbtStack = new Stack<>();
 
-    public PanelScrollingNBT(IGuiRect rect, NBTTagCompound tag, int btnEdit, int btnAdv, int btnInsert, int btnDelete) {
-        this(rect, btnEdit, btnAdv, btnInsert, btnDelete);
+    public PanelScrollingNBT(IGuiRect rect, NBTTagCompound tag, int btnEdit, int btnAdv, int btnUp, int btnDown, int btnInsert, int btnDelete) {
+        this(rect, btnEdit, btnAdv, btnUp, btnDown, btnInsert, btnDelete);
 
         this.setNBT(tag);
     }
 
-    public PanelScrollingNBT(IGuiRect rect, NBTTagList tag, int btnEdit, int btnAdv, int btnInsert, int btnDelete) {
-        this(rect, btnEdit, btnAdv, btnInsert, btnDelete);
+    public PanelScrollingNBT(IGuiRect rect, NBTTagList tag, int btnEdit, int btnAdv, int btnUp, int btnDown, int btnInsert, int btnDelete) {
+        this(rect, btnEdit, btnAdv, btnUp, btnDown, btnInsert, btnDelete);
 
         this.setNBT(tag);
     }
 
-    private PanelScrollingNBT(IGuiRect rect, int btnEdit, int btnAdv, int btnInsert, int btnDelete) {
+    private PanelScrollingNBT(IGuiRect rect, int btnEdit, int btnAdv, int btnUp, int btnDown, int btnInsert, int btnDelete) {
         super(rect);
 
         this.btnEdit = btnEdit;
         this.btnAdv = btnAdv;
+        this.btnUp = btnUp;
+        this.btnDown = btnDown;
```

Comment:
both buttons are active if the length is > 1, even when at the top/bottom and clicking does nothing